public/static/lang/en.json

```diff
@@ -73,6 +73,10 @@
   "Measurement.Details.Websites.HTTP.Request.URL": "URL",
   "Measurement.Details.Websites.HTTP.Response.Body": "Response Body",
   "Measurement.Details.Websites.HTTP.Response.Headers": "Response Headers",
+  "Measurement.CommonDetails.Label.Resolver": "Resolver",
+  "Measurement.CommonDetails.Label.ResolverASN": "Resolver ASN",
+  "Measurement.CommonDetails.Label.ResolverIP": "Resolver IP",
+  "Measurement.CommonDetails.Label.ResolverNetworkName": "Resolver Network Name",
```

Comment:
I have added these keys to the spreadsheet to ensure they are not deleted the next time we sync up the `.json` files.